Pjiwm/kaizominecraft
# Mob randomization

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*
Mobs should have random aspects about them, right now most of them just get a speed or strength boost,
this will be changed with random effects and attribute changes.

**Please provide enough information so that others can review your pull request:**

There needs to be a general random boost and a specific one for certain mobs such as creepers.
The following features will be implemented

- [x] a generic mob buff method that randomizes health attributes and effects
- [x] unique skeleton arrows
- [x] random creeper explosion size
- [x] random charged creepers
- [x] random slime sizes
- [x] super mob with even higher effects and glowing effect
- [x] ghast higher explosions


**Explain the **details** for making this change. What existing problem does the pull request solve?**

We solve predictable mob patterns with randomization, players will never be sure how strong a certain mob is when fighting.

**Test plan (required):**
Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request has visual changes.

1. Run local server
2. Create a redstone clock that summons mobs with a command block.
3. Check if randomization and new features work.

**Closing issues**
closes #19 

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
